src/lwc/geFormWidgetTokenizeCard/geFormWidgetTokenizeCard.js

```diff
@@ -15,6 +15,7 @@ export default class geFormWidgetTokenizeCard extends LightningElement {
     @track domain;
     @track visualforceOrigin;
     @track isLoading = true;
+    @api cardHolderName;
```

Comment:
Requesting that `cardHolderName` actually be an object with `firstName`, `lastName`, and `accountName` as properties instead of a string. Tokenization requires just one string to tokenize (`nameOnCard`), but the purchase call requires both `firstName` and `lastName`.

Not sure how we want to handle this situation when making a purchase call for an Account. We _could_ populate either of those name fields with the Account Name and the other with any string in this scenario, but it seems wrong to do that if the request is requiring a first and last name.

@ptewson-sfdo @vmartinov-sforg @voduyemi

Reply:
@psadie-sfdo I have made updates to return an object with the `firstName`, `lastName` and `accountName`. Let me know if this works for you.

Reply:
@voduyemi - Thanks for making this change! We're good from my perspective regarding this fabricated cardholder name for tokenization. Let's chat about the purchase call end of things for organization accounts with the broader team.